tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -80,8 +80,20 @@ test_spi_lpd8806: lpd8806@9 {
 	spi-max-frequency = <0>;
 };
 
+test_spi_uc8176: uc8176@a {
+	compatible = "ultrachip,uc8176";
+	label = "UC8176";
```

Comment:
Remove this; we are phasing out use of label properties in upstream Zephyr, so let's not add any more, please.

Reply:
Done.